docs/imageMgmtAPI.rst

```diff
@@ -82,6 +83,40 @@ owner (these APIs will be provided later). ``GUEST`` role can have only read/get
 
   Status: 201 Created
   Header -> Location: /imageTypes/{id}
+  
+Update image type (Update)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To update the owners of a image created in the database, we can use the 
+
+- **Method:** POST
+- **Request URL:** /imageTypes
+- **Required Request Parameters:** 
+    session.id={Replace browser session id here}
+    updateImageOwners=addImageOwners/removeImageOwners 
```

Comment:
Do you think it would be too much to have an example for both addition and removal?